planning/development-strategy.md

```diff
@@ -20,6 +20,9 @@
 - [ ] *label* tag named add item
 - [ ] *input* tag type="text" for the user to enter an item
 - [ ] *add* button
+- [ ] *form* to have the ToDo list
```

Comment:
Hey, we already mentioned  an input field in **_for: tasks to be done_** (tag li with input type="checkbox")